.mocharc.json

```diff
@@ -2,5 +2,5 @@
   "extension": ["ts"],
   "spec": ["tests/**/**/**.test.ts"],
   "require": "ts-node/register",
-  "timeout": "20000"
+  "timeout": "30000"
```

Comment:
not sure if this should be changed, but some of the failing tests still showed a 20000ms timeout even after changing the other timeout to 30000ms so I figured that it should be changed here as well

Reply:
i believe [this change](https://github.com/mongodb/docs-search-transport/pull/108/files#diff-d83d3d9cf0be5dce464942d69b396844b0731533be16c4b9edcf44875a23a007R44) should only be for [the test suite](https://mochajs.org/#suite-level) but i am fine with bumping this up in general